packages/terra-button-group/tests/nightwatch/ButtonGroupOnChange.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
```

Comment:
This file seems to be about the same as the SingleSelect, do we really need both of them? Testing the single and multi select implementations seems to be sufficient to test the onChange.

Reply:
Agree and removed: https://github.com/cerner/terra-core/pull/1155/commits/ed6ceb87b5bfd650e1259d62a29f077c6b4ef36f